plugins/package-managers/swiftpm/src/funTest/assets/projects/synthetic/project-with-lockfile/Package.resolved

```diff
@@ -78,8 +78,8 @@
         "repositoryURL": "https://github.com/apple/swift-collections.git",
         "state": {
           "branch": null,
-          "revision": "d029d9d39c87bed85b1c50adee7c41795261a192",
-          "version": "1.0.6"
+          "revision": "94cf62b3ba8d4bed62680a282d4c25f9c63c2efb",
```

Comment:
Hmm. This doesn't really reduce the diff with `Package.resolved` from the last commit, as that also seems to "unnested" the whole structure.

Reply:
It does reduce the diff with the expected output file.

Reply:
The unnesting should have been probably part of the middle commit. I'll check that.